### ISSUE TITLE
Debug Transaction

### ISSUE THREAD

**Issue by bigfootdeveloper** [antoinedc/ethernal-cli#16]
I am using ganache-cli forking BSC network and I am using truffle.  I tried to configure de "transactions tracing" which is suported by ganache, but the trace doesn't appear and no error is shown.

What am I doing wrong?

Thanks

**Comment by antoinedc** [comment]
In the Ganache logs, when you do a write transaction, do you see a "debug_traceTransaction" line?

**Comment by bigfootdeveloper** [comment]
I didn't see.  Is there an option to start it with "debug_traceTransaction"?

I see only this.

<img width="1136" alt="screen" src="https://user-images.githubusercontent.com/86575184/134799803-5ce41652-33b5-47f0-af86-1ff78e8e1795.png">

**Comment by antoinedc** [comment]
Ok, in your settings, the tracing option should be set to "Other", and after changing it, you need to restart the Ethernal CLI. Can you double check that this the case, please?

**Comment by bigfootdeveloper** [comment]
Yes, I did it.  I attach an image.  I restarted too the "ethernal" listen and refresh the page.

<img width="478" alt="screen2" src="https://user-images.githubusercontent.com/86575184/134799910-dc922ca7-8f07-4645-8cfa-05aa8c34fd6b.png">

**Comment by antoinedc** [comment]
Yes that's correct, I will take a look to see if I can reproduce the issue, I'll let you know

**Comment by bigfootdeveloper** [comment]
I don't want that you loose your time.  May be this option is not supported in ganache-cli.

Take a look

https://giters.com/trufflesuite/ganache/issues/949

But in it's web appears in the API:

https://www.npmjs.com/package/ganache-cli

**Comment by antoinedc** [comment]
What's not supported is one of the options, but the command itself is.

No worries, if it doesn't work it definitely needs to be fixed anyway!
